css/excercise_one/excercise_one.css

```
@@ -0,0 +1,65 @@
+* {
+  font-family: 'Poppins';
+  color: black;
+  background-color: white;
+  text-decoration: none;
+}
+
+nav{
```

space after nav (fix the rest as well please), also styles go in alphabetical order, please reorder all of them. example:
```suggestion
nav {
  align-items: center;
  display: flex;
  justify-content: space-between;
  margin: 25px;
}
```